src/main/java/org/stellar/sdk/TransactionBuilder.java

```diff
@@ -57,6 +59,41 @@ public TransactionBuilder(TransactionBuilderAccount sourceAccount, Network netwo
     this(AccountConverter.enableMuxed(), sourceAccount, network);
   }
 
+  /**
+   * Construct a new transaction builder from a transaction.
+   *
+   * @param transaction the transaction to rebuild
+   */
+  public TransactionBuilder(Transaction transaction) {
```

Comment:
nice, this builder pattern is good way to maintain immutability and convenience!